src/Sylius/Bundle/AdminBundle/Resources/private/js/sylius-select-attribute.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+(function ($) {
+    'use strict';
+
+    $.fn.extend({
+        handleSelectAttributeTranslatableValues: function () {
+            var element = $(this);
+
+            element.find('[data-form-collection="add"]').on('click', function() {
+                setTimeout(function() {
```

Comment:
What is the benefit of waiting 100ms? Or why doesn't it work without I mean..

Reply:
If we listen on "click" of the **Add** button, it takes some miliseconds to add this value, triggering `accordion()` immediately don't catch the latest element. As I said, I'm not a JS-guy at all, I just know that it helps 😄 I'll appreciate any better solution.

Reply:
Probably listening to the adding of the element is 'more clean' (https://stackoverflow.com/questions/18746825/adding-an-event-to-catch-when-a-list-item-is-added-to-a-select-jquery), but this also does the job probably :-D

Reply:
@stefandoorn this `DOMNodeInserted` trick did the job! Thanks 😄